src/os/bluestore/bluestore_types.h

```diff
@@ -701,9 +700,8 @@ struct bluestore_blob_t {
     uint64_t blob_len = get_logical_length();
     assert((blob_len % (sizeof(unused)*8)) == 0);
     assert(offset + length <= blob_len);
-    uint64_t chunk_size = blob_len / (sizeof(unused)*8);
-    uint64_t start = ROUND_UP_TO(offset, chunk_size) / chunk_size;
-    uint64_t end = (offset + length) / chunk_size;
```

Comment:
This one is different than the others.  @ifed01 is that a bug?

Reply:
No, add_unused bypasses partially overlapped chunks contrary to other methods that include them